src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -12,19 +12,25 @@
 using Aspire.Dashboard.Otlp.Grpc;
 using Aspire.Dashboard.Otlp.Storage;
 using Microsoft.AspNetCore.Authentication.Certificate;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 using Microsoft.AspNetCore.Server.Kestrel.Https;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.FluentUI.AspNetCore.Components;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
 
 namespace Aspire.Dashboard;
 
 public class DashboardWebApplication : IAsyncDisposable
 {
-    internal const string DashboardInsecureAllowAnonymousVariableName = "DOTNET_DASHBOARD_INSECURE_ALLOW_ANONYMOUS";
```

Comment:
I still want this config key to set frontend and OTLP auth modes to Unsecured. The most common usage of the dashboard image will be someone playing around with it on their local machine. They won't want to use auth.

This is a command line/env var friendly way (no double underscore escape) to set both values to unsecured.

Reply:
I mentioned on another PR that having a single kill switch for all security measures means that users won't get the opportunity to knowingly opt-out of future security features. We want each to be opted out individually to prevent that.

Very few users will need to ever set these config keys. Those who do will be guided by documentation and helpful error messages. The safety outweighs the convenience factor here.

Reply:
I'll contact experts offline.